requirements.txt

```diff
@@ -1,91 +1,53 @@
+-c https://raw.githubusercontent.com/ska-sa/katsdpdockerbase/master/docker-base-build/base-requirements.txt
+
 addict==2.1.1
-aioconsole                 # via aiomonitor
 aiohttp
 aiohttp-jinja2==1.4.2
-aiohttp-retry              # via katsdpmodels
 aiokatcp
-aiomonitor
 aioredis
 aiozk==0.14.0
-astropy                    # via katsdpmodels
 async-timeout
-attrs                      # via aiohttp, prometheus_async
-certifi                    # via requests
-chardet                    # via aiohttp, requests
-cityhash                   # via katdal
 click==7.0                 # via Flask
 dash==1.0.2
 dash-core-components==1.0.0
 dash-html-components==1.0.0
 dash-table==4.0.2
 dash-dangerously-set-inner-html==0.0.2
 dash-renderer==1.0.0       # via dash
-dask                       # via katdal
 decorator
 docker-pycreds==0.3.0      # via docker
 docker==3.4.1
-ephem                      # via pyephem
 flask==1.1.1               # via dash
 flask-compress==1.4.0      # via dash
-future                     # via katdal
-h5py                       # via katdal, katsdpmodels
-hiredis
+hiredis                    # speeds up katsdptelstate
 http-parser==0.9.0         # via pymesos
-idna                       # via requests
-idna-ssl                   # via aiohttp
-importlib-metadata         # via jsonschema
 ipython-genutils==0.2.0    # via nbformat
 itsdangerous==1.1.0        # via flask
-jinja2                     # via aiohttp-jinja2
+jinja2
 jsonschema
 jupyter-core==4.4.0        # via nbformat
 katportalclient
-katversion
 kazoo==2.8.0
-llvmlite                   # via numba
-markupsafe                 # via jinja2
 monotonic==1.5             # via prometheus_async
-msgpack                    # via katsdptelstate
-multidict                  # via aiohttp, yarl
 nbformat==4.4.0            # via plotly
-numba                      # via katdal
 netifaces
 networkx==2.0
 numpy
-omnijson                   # via katportalclient
 plotly==4.0.0              # via plotly
```

Comment:
Plotly is not listed as a direct dependency, and it arrives "via plotly"... 🙂

Reply:
Heh, requirement-ception :-) Fixed the comment.